tools/scylla-sstable.cc

```
@@ -2889,7 +2894,11 @@ Validate the specified sstables:
 
         std::vector<sstables::shared_sstable> sstables;
         if (app_config.count("sstables")) {
-            sstables = load_sstables(schema, sst_man, app_config["sstables"].as<std::vector<sstring>>());
+            try {
+                sstables = load_sstables(schema, sst_man, app_config["sstables"].as<std::vector<sstring>>());
+            } catch (...) {
+                return 1;
```

... print all exceptions here. There can be exceptions from other parts, not just load() and we will now silently swallow those.

---

But parallel for each will propagate only one of them

---

we can log an error at both levels

---

Ok, lets do that then.

---

done in v2